### PR TITLE
bump: :completion vertico

### PR DESCRIPTION
**Pull request by iyefrat**
This PR bumps the vertico module, switches to use consult-fd now that it's upstream instead of re implementing it, and adapts to an upstream variable rename that happened way back that slipped by at the time

### CODE REVIEW

File: modules/completion/vertico/packages.el [hunk 1]
```diff
@@ -1,22 +1,19 @@
 ;; -*- no-byte-compile: t; -*-
 ;;; completion/vertico/packages.el
 
-(package! vertico
-  :recipe (:host github :repo "minad/vertico"
-           :files ("*.el" "extensions/*.el"))
-  :pin "a28370d07f35c5387c7a9ec2e5b67f0d4598058d")
+(package! vertico  :pin "03bfb71a2b5db296a93eeb0752bd934d112e0792")
```
Comment (iyefrat) [inline modules/completion/vertico/packages.el:4]: vertico was recently added to melpa, so no need for this workaround to have the commit history

File: modules/completion/vertico/config.el [hunk 2]
```diff
@@ -304,9 +299,9 @@ orderless."
   (map! (:map embark-file-map
          :desc "Open target with sudo"        "s"   #'doom/sudo-find-file
          (:when (modulep! :tools magit)
-          :desc "Open magit-status of target" "g"   #'+vertico/embark-magit-status)
+           :desc "Open magit-status of target" "g"   #'+vertico/embark-magit-status)
          (:when (modulep! :ui workspaces)
-          :desc "Open in new workspace"       "TAB" #'+vertico/embark-open-in-new-workspace))))
+           :desc "Open in new workspace"       "TAB" #'+vertico/embark-open-in-new-workspace))))
```
Comment (iyefrat) [inline modules/completion/vertico/config.el:304]: this change is due to auto-formatting the file. seems that in other areas of the code `:when` arguments are sometimes indented and sometimes not, but in where they are not `SPC c f` changes the formatting. Has this changed recently?

Comment (iyefrat) [reply modules/completion/vertico/config.el:304]: oh lol there was the format PR, probably due to that. still, seems to imply that a lot of the doom codebase as is isn't formatted (or i have some formatting misconfiguration). @elken do you know anything about this?

Comment (elken) [reply modules/completion/vertico/config.el:304]: Yeah Henrik and I were talking a while ago about writing an improved elisp formatter for us to dogfood, you'd have to ask him where that got to

**Comment by hlissner** [comment]
Sorry for taking so long to get to this. @iyefrat Would you mind rebumping this module (assuming it's merge-ready otherwise)?

File: modules/completion/vertico/config.el [hunk 3]
```diff
@@ -148,13 +145,11 @@ orderless."
...
-    (setq +vertico-consult-fd-args
-          (if doom-projectile-fd-binary
-              (format "%s --color=never -i -H -E .git --regex %s"
-                      doom-projectile-fd-binary
-                      (if IS-WINDOWS "--path-separator=/" ""))
-            consult-find-args)))
+  (if doom-projectile-fd-binary
+      (setq consult-fd-args
+            '(doom-projectile-fd-binary
+              "--full-path --color=never -i -H -E .git --regex"
```
[the remark this answers is not in the document]
Comment (iyefrat) [reply modules/completion/vertico/config.el]: thanks! added with minimal changes

File: modules/completion/vertico/config.el (hunk 2 above)
Comment (hlissner) [reply modules/completion/vertico/config.el:308]: @hpfr This has me stumped too. I've been unable to reproduce it in my quick tests (i.e. `indent-region` respects the `indent-plists-as-data` property, and on Emacs HEAD). My local version of Doom is somewhat different than what's on `master` though, so I'll set up a better sandbox to try again in later this week.

**Review by hlissner — APPROVED** [review]

**Comment by hlissner** [comment]
Sorry for taking so long to get to this, but thanks again for your help!

**Pull request merged**